airbyte-ci/connectors/pipelines/pipelines/builds/__init__.py

```diff
@@ -50,9 +52,26 @@ async def run_connector_build_pipeline(context: ConnectorContext, semaphore: any
         async with context:
             build_result = await run_connector_build(context)
             step_results.append(build_result)
-            if context.is_local and build_result.status is common.StepStatus.SUCCESS:
-                connector_to_load_to_local_docker_host = build_result.output_artifact[LOCAL_BUILD_PLATFORM]
-                load_image_result = await common.LoadContainerToLocalDockerHost(context, connector_to_load_to_local_docker_host).run()
-                step_results.append(load_image_result)
             context.report = ConnectorReport(context, step_results, name="BUILD RESULTS")
         return context.report
+    
+async def run_cdk_build(context: CDKContext) -> StepResult:
+    """Run a build pipeline for a single cdk."""
+    return await java_cdk.run_cdk_build(context)
+
+
+async def run_cdk_build_pipeline(context: CDKContext) ->  Optional[ConnectorReport]:
+    """Run a build pipeline for a single cdk.
+
+    Args:
+        context (CDKContext): The initialized connector context.
+
+    Returns:
+        None
+    """
+    step_results = []
+    async with Connection(Config(log_output=None, execute_timeout=None)) as dagger_client:
+        context.dagger_client = dagger_client.pipeline("Java CDK Build")
+        build_result = await run_cdk_build(context)
+        step_results.append(build_result)
+    return None
```

Comment:
oh nice, i had thought that only ConnectorReport existed and not just Report, maybe reporting is not going to be as difficult as I had thought it would be.